chromium_src/components/variations/variations_seed_store.cc

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#define BRAVE_K_PUBLIC_KEY \
+  0x30, 0x59, 0x30, 0x13, 0x06, 0x07, 0x2a, 0x86, 0x48, 0xce, 0x3d, 0x02, 0x01, \
+  0x06, 0x08, 0x2a, 0x86, 0x48, 0xce, 0x3d, 0x03, 0x01, 0x07, 0x03, 0x42, 0x00, \
+  0x04, 0xfd, 0x94, 0x51, 0xd4, 0xab, 0x28, 0xda, 0x02, 0x6a, 0xb8, 0xb3, 0x16, \
+  0xce, 0x1e, 0x09, 0x49, 0xa8, 0xa3, 0x03, 0x8a, 0x92, 0x9b, 0x59, 0xa9, 0x9e, \
+  0x43, 0xe8, 0xdb, 0x76, 0x81, 0xf0, 0x44, 0xfc, 0xf1, 0x93, 0x77, 0xf7, 0xd6, \
+  0xba, 0xe8, 0x6a, 0x46, 0xcd, 0x8a, 0xa7, 0xff, 0x73, 0x32, 0x4b, 0x0b, 0x83, \
+  0x3b, 0x16, 0x03, 0x71, 0x8b, 0x58, 0x0d, 0x3c, 0xa2, 0x55, 0x37, 0x43, 0x9a, \
+}; \
```

Comment:
does this correspond to the private key in https://github.com/brave/brave-variations/commit/a57181e2dfd83f5bc339eb8064883433686a9bbb?

if so please rotate it and put the new private key in 1password developers vault

we try to avoid having any secrets in github even in private repos.

Reply:
@diracdeltas no, it is patching the public key in https://source.chromium.org/chromium/chromium/src/+/master:components/variations/variations_seed_store.cc;l=37 with https://github.com/brave/brave-variations/blob/main/publickey for signature verification

Reply:
@moritzhaller i understand, but i'm asking if the private key that corresponds to the public key is the one in the brave-variations repo. i know it's a private repo but we still want to avoid having secrets in there because it's too easy to push a private repo accidentally.

Reply:
@diracdeltas gotcha, will rotate and purge!